### PR TITLE
Base layout added

### PR DESCRIPTION
**Pull request by shangan23**
Material UI 
- Header
- Footer
- Cart Icon
- Content Area

### CODE REVIEW

**Comment by ramsunvtech** [comment]
@shangan23 can you pull and rebase to avoid the 3rd commit ?

image

File: src/app/app.component.ts [hunk 1]
```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'cake-shop-ng-app';
+  title = 'Shopping Cart';
```
Comment (ramsunvtech) [inline src/app/app.component.ts:9]: lets keep the Cake Shop

File: src/app/components/common/footer/footer.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<mat-toolbar color="secondary">
+  <span class="mat-small">@copyright 2020</span>
```
Comment (ramsunvtech) [inline src/app/components/common/footer/footer.component.html:2]: can we add localization module in next PR

https://angular.io/guide/i18n#localizing-your-app

**Comment by shangan23** [comment]
> @shangan23 can you pull and rebase to avoid the 3rd commit ?
> 
> image

Sure, this time I need your help. Actually, I got confused

File: src/app/components/common/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<h1 class="mat-display-1">
+    Welcome to Shopping cart Angular Application.
```
Comment (ramsunvtech) [inline src/app/components/common/home/home.component.html:2]: keep as `Welcome to Cake Shop`

File: src/app/modules/shared.module.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+@NgModule({
+  declarations: [
+    HomeComponent,
+    HeaderComponent,
+    FooterComponent,
+    NotFoundComponent
+  ],
+  imports: [
+    CommonModule,
+    MatGridListModule,
```
Comment (ramsunvtech) [inline src/app/modules/shared.module.ts:22]: keep the export name, for example from `material/grid-list` to `MuiGridList`

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

**Pull request merged**